### PR TITLE
 Only use viewport height to separate live blog adverts

### PR DESCRIPTION
**Pull request by rich-nguyen**
We no longer pass `index`, and it seems sensible to keep the logic solely based on viewport.

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/liveblog-adverts.js [hunk 1]
```diff
@@ -7,24 +12,12 @@ import { commercialFeatures } from 'common/modules/commercial/commercial-feature
 import { createSlot } from 'commercial/modules/dfp/create-slot';
 import { spaceFiller } from 'common/modules/article/space-filler';
 
-const INTERVAL = 5; // number of posts between ads
 const OFFSET = 1.5; // ratio of the screen height from which ads are loaded
 const MAX_ADS = 8; // maximum number of ads to display
 
-type spaceFillerRules = {
```
Comment (LATaylor-guardian) [inline static/src/javascripts/projects/commercial/modules/liveblog-adverts.js]: Where did these go? Were they duplicated elsewhere?

Comment (rich-nguyen) [reply static/src/javascripts/projects/commercial/modules/liveblog-adverts.js]: yes, I had previously worked on making a single type for rules. But live blogs had always duplicated their own rules type.

**Review by kelvin-chappell — APPROVED** [review]
Looks good to me 👍 
but may need a second opinion

**Comment by LATaylor-guardian** [comment]
👍

**Pull request merged**